### ISSUE TITLE
Skip to main content link is not present in the user pages

### ISSUE THREAD

**Issue by reetagithub** [DSpace/dspace-angular#2523]
**Is your feature request related to a problem? Please describe.**
This is an accessibility issue, an important feature is missing from DSpace7. When using a screen reader, the user should have the opportunity to skip all the stuff in the upper part of the screen (navigation, login, logo etc.) and get to the main contents of the page. This is a requirement in the accessibility legislation in the EU which is based on WCAG 2.2 levels A and AA. This More info here ([Bypass Blocks](https://www.w3.org/TR/WCAG22/#bypass-blocks)) and here [Webaim](https://webaim.org/techniques/skipnav/). 

**Describe the solution you'd like**
In the beginning of each page there should be a hidden link that leads to the main content of the page and skips the irrelevant content. You can see a way to do this right on [WebAim's page](https://webaim.org/) and its [source code](view-source:https://webaim.org/) ('skiptocontent')

**Describe alternatives or workarounds you've considered**
There is no workaround. This feature is crucial for the users with screen readers, and it is part of accessibility requirements. If this is not made, it is very difficult for vision impaired users to explore the pages.

**Comment by reetagithub** [comment]
Thank you! I am sure you will add the text to translation file as well.

**Comment by tdonohue** [comment]
@reetagithub : Yes, the merged PR already makes it translatable.  See the updates to the `en.json5` file: https://github.com/DSpace/dspace-angular/pull/2607/files#diff-63dcd993feb69f5096737902df76b82a423ebd1acc78cc9b37798b2c05a5436f

**Comment by reetagithub** [comment]
Perfect!

I hope that the texts to be translated will be put to the files, especially fi.json5. I tried to update it myself previously this year, and it became a mess, so now I will wait for some one else  (who knows how to) to update the translation files.

Thank you, Tim!

Tim Donohue ***@***.***> kirjoitti 13.11.2023 kello 17.19:



@reetagithub<https://github.com/reetagithub> : Yes, the merged PR already makes it translatable. See the updates to the en.json5 file: https://github.com/DSpace/dspace-angular/pull/2607/files#diff-63dcd993feb69f5096737902df76b82a423ebd1acc78cc9b37798b2c05a5436f

**Comment by anis-moubarik** [comment]
This feature doesn't seem to work on Firefox, I can access the button, but clicking it does nothing.

Also with Chrome and Firefox, while logged in, the button is obstructed by the left side panel.

**Comment by alexandrevryghem** [comment]
@anis-moubarik: I just tested it again for Firefox & Chrome and clicking on it & hitting enter on it does work. The next time you click on the tab the focus will be on the `leading institutions using DSpace` url and not to admin sidebar/header. When you are logged in I can indeed see that the admin sidebar is displayed over it, we should probably check if this is still happening after #2676 is merged in

**Comment by anis-moubarik** [comment]
@alexandrevryghem You're right! I'm used to the viewport centering the content with jump to content -links (like it seems to do with Chrome natively). I'll probably add scrollIntoView call in the skipToMainContent function.
